### PR TITLE
style: dashboard skeleton

### LINKED ISSUE #5: Dashboard UI
- [x] harus ada informasi tentang logged in user
- [x] harus ada tombol untuk navigasi ke /dashboard/create untuk create event
- [x] harus ada list event yang user miliki (mockup)
- [x] harus ada tombol untuk logout

tolong di centang jika sudah dikerjakan

semua data menggunakan data dummy

### PR DESCRIPTION
**Pull request by ilmi**
I've add dashboard skeleton (no styling)

### CODE REVIEW

File: src/components/dashboard/components/DashboardPage.jsx [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+            </p>
+          </div>
+          <div>
+            <p className="text-sm text-gray-600">
+              Organized by : Mafatikhul Ilmi
+            </p>
+            <p className="text-sm text-gray-600">
+              Show Time : Sunday, 4 Feb 2024
+            </p>
+          </div>
```
Comment (alfonsusac) [inline src/components/dashboard/components/DashboardPage.jsx]: Great Job.

Please, Use array of objects and .map instead of hard coding the htmls.

**Review by alfonsusac — CHANGES_REQUESTED** [review]
Also please attach screenshot of the result

File: src/components/dashboard/components/DashboardPage.jsx (hunk 1 above)
Comment (ilmi) [reply src/components/dashboard/components/DashboardPage.jsx]: image

**Comment by alfonsusac** [comment]
#5

**Review by alfonsusac — APPROVED** [review]

**Pull request merged**